content/faq.md

```diff
@@ -40,48 +46,36 @@ title = "Frequently Asked Questions (FAQ)"
+> VCMI follows original Heroes 3 mechanics as close as possible and should be fully compatible with it. However, some obvious bugs were fixed, while some features were made optional. The AI is brand new and completely different.
 
 * Higher resolution / random map generator is not working!
 
-> Get "VCMI essential files mods (1.01)" from <https://wiki.vcmi.eu/Mod_list>.
+> Get "VCMI Extras" mod from our launcher. If you are on Android - as of VCMI 1.1.0 this feature is not yet present in Android launcher as it uses different code than other platforms, but is in late development stage and hopefully will be there soon. Workaround while it's not available would be downloading mod on PC and moving Mods folder to android to same folder that has Heroes 3 data.
 
 * Game screen resolution is glitchy! Mouse goes beyond window / fullscreen is not properly stretched!
 
-> On PC: Try real fullscreen mode, you can activate it in VCMI launcher. On Android: Glitchy screen borders is known bug.
+> Situation written here is accurate as of VCMI 1.1.0 - On PC: Try real fullscreen mode, you can activate it in VCMI launcher. If it happens on mobile platform then it may be yet unfixed bug, if changing resolution won't fix it.
 
 * Is there any way to boost adventure AI? I want harder opponent.
 
-> Yeah, the easiest thing to try is to use cheats for resources and/or full map reveal, using parameter that specifies target players. Typing 'ai' after cheat name causes only AI players to be affected by it, player color names are also recognized. So you can for example type 'vcmiformenos ai' to give extra resources to AI player, 'vcmieagles blue' to reveal full map for blue player etc.  Note that these cheats do not guarantee any improvement, for example revealing whole map may cause AI to overly focus on far goals, missing local details. Distance is one of factors when deciding what to do, but still. More advanced thing you can do is experimenting with AI Values for various objects / object classes by changing AI Value (exact object value) and Default AI Value (object class value) in object config JSON files. Range of effective AI Value is 0-20000. Also it's important to know that AI Values are tied to saved games by design, so you need to start new game to test effectiveness of new AI Values.
+> As of VCMI 1.1.0 the default adventure map AI algorithm is VCAI. Nullkiller AI is alternative, more recent AI that you can choose in VCMI launcher, that is generally more challenging, but may be slow on big maps. Ability to swap AI is new feature in Android version of game since VCMI 1.1.0 so if you played on older version it's worth checking.
```

Comment:
```suggestion
> As of VCMI 1.1.0 the default adventure map AI algorithm is VCAI. Nullkiller AI is alternative, more recent AI that you can choose in VCMI launcher settings. That is generally more challenging, if you want more, check out  Resourceful AI mod, which provides options for empowering AI and to speed up it. Ability to swap AI is new feature in Android version of game since VCMI 1.1.0 so if you played on older version it's worth to check it out.
```